docs/manifest.json

```diff
@@ -319,6 +318,30 @@
           "path": "./contributing/SECURITY.md"
         }
       ]
+    },
+    {
+      "title": "API",
+      "description": "Learn how to use Coderd API",
```

Comment:
I think outside of the source code, `coderd` doesn't mean much.

```suggestion
      "description": "Learn how to use the Coder API",
```

Reply:
Fixed